modules/ROOT/pages/install-rtfctl.adoc

```diff
@@ -52,70 +52,534 @@ curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -
-| upgrade-cluster | Upgrades the Runtime Fabric cluster.
-| validate | Validate cluster for Runtime Fabric installation.
-| version | Displays version information.
-|=== 
+|xref:apply[`apply`] | Applies changes to a Runtime Fabric configuration
+|xref:update[`update`] | Updates rtfctl
+|xref:wait[`wait`] | Waits for Runtime Fabric components to become ready
+|===
+
+=== `apply` 
+
+Applies changes to a Runtime Fabric configuration.
+
+* Applies a HTTP proxy configuration
++
+----
+rtfctl apply http-proxy http://<domain>:<port>
+----
+
+* Applies a HTTP proxy configuration with hostnames bypassing the proxy
++
+----
+rtfctl apply http-proxy http://<domain>:<port> --no-proxy "DOMAIN.com,DOMAIN2.com"
+----
+
+* Applies a HTTP proxy configuration with authentication
++
+----
+rtfctl apply http-proxy http://username:password@<domain>.com:<port>
+----
+
+* Clears an HTTP proxy configuration
++
+----
+rtfctl apply http-proxy "" --no-proxy ""
+----
+
+* Applies an ingress configMap for a specific pod 
++
+[NOTE]
+--
+For Runtime Fabric on Self-Managed Kubernertes versions 1.8 and 1.9 only
+--
++
+----
+rtfctl apply ingress-configmap "/<pod>/<path-to-configMap.yaml>"`
+----
+
+* Applies a secure property
++
+----
+rtfctl apply secure-property --key <key> --value <value> -n <environment>
+----
+
+* Applies a system configuration on a host node
++
+----
+rtfctl apply system-configuration
+----
+
+* Applies a system configuration on a host node without downloading the latest scripts
++
+----
+rtfctl apply system-configuration --skip-download
+----
+
+* Applies a Mule license
++
+----
+rtfctl apply mule-license '<BASE64-encoded-license>'
+----
+* Applies a monitoring proxy
++
+----
+rtfctl apply monitoring-proxy "socks5://<username>:<password>@<monitoring-URL>:<PORT>"`
+----
+
+* Clears a monitoring proxy configuration
++
+----
+rtfctl apply monitoring-proxy ""
+----
+
+=== `update` 
+
+Updates `rtfctl`.
+
+* Updates from the US control plane
++
+----
+rtfctl update
+----
+
+* Updates `rtfctl` from the EU control plane
++
+----
+rtfctl update --host eu1.anypoint.mulesoft.com
+----
+
+=== `wait`
+Waits for Runtime Fabric components to become ready
+
+* Specifies the number of seconds to wait 
+----
+rtfctl wait <value in seconds>
+----
+
+* Specifies the Number of seconds to pass before timing out the wait (Default is 600 seconds)
+
+----
+rtfctl wait --timeout <value in seconds>
+----
+
+== Commands for Installing and Upgrading Runtime Fabric
+
+Use the following commands when installing or upgrading Runtime Fabric, or when performing a back up or restore. 
 
 [%header%autowidth.spread,cols="a,a"]
-.rtfctl Flags
 |===
-|Flag |Description
-| -h, --help | Displays `rtfctl` help information.
-| -n, --namespace string | If present, displays the namespace scope.
-| --pod string | If present, displays the pod name.
+|Command |Description
+|xref:install[`install`] | Installs Runtime Fabric
+|xref:reinstall[`reinstall`] | Re-installs Runtime Fabric components
```

Comment:
lets remove this section - that command should actually be a subcommand under `appliance`